.github/workflows/update-release-links.yml

```diff
@@ -0,0 +1,19 @@
+name: Update release links
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+
+jobs:
+  update-release-links:
+    if: github.repository == 'nodejs/node'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger update-links workflow on nodejs/release-cloudflare-worker
+        run: |
+          gh workflow run update-links.yml --repo nodejs/release-cloudflare-worker
```

Comment:
We should also trigger the job (after this one) to deploy the worker to prod 👀

Reply:
I think if you want this to happen, it should be done from the nodejs/release-cloudflare-worker repo.

Reply:
Right, was pondering on that. @flakey5 it might make sense to make it autodeploz after updating the links

Reply:
it currently opens a PR and needs a manual approval & merge - we will need to change it to commit and push to main - I dont know if that is possilbe

Reply:
> it currently opens a PR and needs manual approval & merge - we will need to change it to commit and push to main - I don't know if that is possible

It is. You can make the workflow commit and push to main directly,

Reply:
I'd definitely be for either letting the releasers merge the pr it generates in release-cloudflare-worker or having them automatically merged and then deployed. I'm not sure exactly how we could get the second option working right now though

Reply:
It is easy to do the 2nd. Just instead of opening a PR simply do a commit and then push. You can see a workflow that does similar on the nodejs.org repository (I think the i18n workflow)

Reply:
Yeah that sounds good to me

Reply:
Should we create an issue on the worker repo?

Reply:
nodejs/release-cloudflare-worker#66